code/modules/cargo/packs.dm

```diff
@@ -1408,6 +1408,51 @@
 	crate_name = "plasma assembly crate"
 	crate_type = /obj/structure/closet/crate/secure/plasma
 
+/datum/supply_pack/science/raw_flux_anomaly
+	name = "Raw Flux Anomaly"
+	desc = "The raw core of a flux anomaly, ready to be implosion-compressed into a powerful artifact."
```

Comment:
description of these should probably mention them being access locked to toxins storage